fastestimator/op/tensorop/augmentation/cutmix_batch.py

```diff
@@ -53,14 +53,13 @@ class CutMixBatch(TensorOp):
         AssertionError: If the provided inputs are invalid.
     """
     def __init__(self,
-                 inputs: str,
+                 inputs: Iterable[str],
```

Comment:
The doc string for inputs and outputs needs to be updated. it would also be good to add an assert validating that exactly 2 keys are provided in each case